### PR TITLE
Improvements to Footer, APIs, top 20 docs

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

*Improves:
* footer content. Social media icons section broke with the Docusaurus v3 update.
* API content.
* Fixes formatting for top 20 most viewed docs.

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

File: docs/api/beta.md [hunk 1]
```diff
@@ -30,47 +30,16 @@ Our APIs are built with [OpenAPI](https://www.openapis.org/). You can generate c
```
Comment (jpipkin1) [inline docs/api/beta.md:30]: Why is this article in beta? It's been that way for as long as I've been here. Just curious.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce. I see you snuck in a little contractions cleanup!

**Comment by kimsauce** [comment]
> Looks good, @kimsauce. I see you snuck in a little contractions cleanup!

Yes! Some docs were breaking [our own rule](https://help.sumologic.com/docs/contributing/style-guide/#downloadable-files) about negative contractions 😂

File: docs/api/beta.md (hunk 1 above)
Comment (kimsauce) [reply docs/api/beta.md:30]: Great catch! Most of them still actually are in beta, but I'll update the ones that GA'd and then resubmit this PR.

Comment (jpipkin1) [reply docs/api/beta.md:30]: This one should stay in beta:
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes/

GA is projected to be April 30.

**Pull request merged**